parser/parser.go

```diff
@@ -0,0 +1,253 @@
+// go run parser.go [file_path] [semster]
+// example: go run parser.go "Fall 2019.csv" 19F
+
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+
+	"parser/configs"
```

Comment:
I don't think these input statements are correct - I think if you delete them and run goimports on the file it should fill it in for you

Reply:
You'll probably have you manually add in configs though from 'github.com/utdnebula/nebula-api/api/configs'

Reply:
Here's the error I'm getting:
`parser.go:23:2: package parser/configs is not in GOROOT (/usr/lib/go/src/parser/configs)`